docs/additional-deployment-options/cmr_launchpad_authentication.md

```diff
@@ -17,17 +17,17 @@ Steps for setting up the CMR Client to use Launchpad authentication can be found
 1. Upload PKI certificate to S3:
 
     - Upload the PKI certificate `.pfx` file to S3
-    - Use `{{system_bucket}}` as bucket name and `{{prefix}}/crypto/launchpad.pfx` as key
-      - `{{system_bucket}}` and `{{prefix}}` are configured in `app/config.yml`.
-      - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad` configuration in `app/config.yml`.
+    - Use `system_bucket` as bucket name and `<prefix>/crypto/launchpad.pfx` as key
+      - `system_bucket` and `<prefix>` are configured in `cumulus-tf/terraform.tfvars`.
+      - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad_certificate` configuration in `cumulus-tf/terraform.tfvars`.
 
-2. Provide passphrase of PKI certificate in `app/.env`
+2. Provide passphrase of PKI certificate in `cumulus-tf/terraform.tfvars`
 
     ```text
-      LAUNCHPAD_PASSPHRASE=<LAUNCHPAD_PASSPHRASE>
+      launchpad_passphrase=<LAUNCHPAD_PASSPHRASE>
     ```
 
-3. Configure the `launchpad` configuration parameters in `app/config.yml`.
+3. Configure the `launchpad` configuration parameters in `cumulus-tf/terraform.tfvars`.
 
-   - `cmr.oauthProvider` in `app/config.yml` **must be set to `launchpad` to use Launchpad authentication**.
-   - Example configuration of the `launchpad` can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/app/config.yml)_
+   - `cmr_oauth_provider` in `cumulus-tf/terraform.tfvars` **must be set to `launchpad` to use Launchpad authentication**.
+   - Example configuration of the `launchpad` can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/deployment/sandbox.tfvars)_
```

Comment:
```suggestion
   - Example configuration of other variables necessary for Launchpad integration can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/deployment/sandbox.tfvars)_
```

Reply:
`sandbox.tfvars` only has some of the variables needed for Launchpad integration (the ones not already highlighted in this document), so I think that is worth clarifying